### PR TITLE
Update the index page for OpenTK 4.x

### PR DESCRIPTION
**Pull request by seanofw**
This is a substantial rewrite for the website's index page that brings it up-to-date to the current project status, as of summer 2021.  I don't expect this PR to be accepted as-is, but it's a good place to collect comments and revisions so that we can make the website a bit more current.

Please feel free to suggest changes; this was my best attempt to revise the content on the front page of the website, but it's by no means final.

### CODE REVIEW

**Review by frederikja163 — APPROVED** [review]
This looks good to me so far, i don't really have anything else i think should be added either.

File: index.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
 
-OpenTK is not a game engine, nor does it try to be one. It offers a useful set of libraries, and leaves it up to the developer to make the decisions on how to use them.
+### Current Status
 
-It is the ideal starting point if you want to write a game or scientific application from the ground up.
+- **OpenTK 4.x** is the current, stable version.  It targets .NET Core 3.x and .NET 5+.
+   - [OpenTK 4.x](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/v/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/dt/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/)
+   - [OpenTK 4.x GLWpfControl](https://www.nuget.org/packages/OpenTK.GLWpfControl/) (beta WPF control)
+   - [OpenTK 4.x GLControl](https://www.nuget.org/packages/OpenTK.WinForms) (beta WinForms control)
+- **OpenTK 3.x** is the legacy version.  It targets .NET Framework 4.x.
```
Comment (NogginBops) [inline index.md:15]: It feels weird calling OpenTK 3 a legacy version as we are still merging PRs for it and releasing new versions. 
At least noting that people are welcome to submit OpenTK 3 PRs (and issues), and that we can release new versions is something we want.

File: index.md [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
+   - [OpenTK 3.x](https://www.nuget.org/packages/OpenTK/3.3.1)
+   - [OpenTK GLControl 3.x](https://www.nuget.org/packages/OpenTK.GLControl/) [![NuGet](https://img.shields.io/nuget/v/OpenTK.GLControl.svg)](https://www.nuget.org/packages/OpenTK.GLControl/) [![NuGet](https://img.shields.io/nuget/dt/OpenTK.GLControl.svg)](https://www.nuget.org/packages/OpenTK.GLControl/)
+- OpenTK 5.0, the next major revision of OpenTK, [is in development](https://github.com/opentk/opentk/issues/1207) and is likely to be released later in 2021.  We would welcome any [contributions](https://github.com/opentk/opentk/)!
 
-### Project Status (25/10/2018)
-Work is well underway on OpenTK 4.0, which targets .NET Standard/Core. We would welcome any [contributions](https://github.com/opentk/opentk/)!
+### Help and Support
 
+If you have a question or want to know how to get started, check our extensive [FAQ](faq.html), or try [our tutorials](https://opentk.net/learn/index.html).
```
Comment (NogginBops) [inline index.md:22]: The tutorial link goes to the website tutorials which unfortunately are still OpenTK 3 tutorials, linking to https://github.com/opentk/LearnOpenTK and explaining a bit of the tutorial situation is probably a good idea to do here.

**Review by NogginBops — COMMENTED** [review]
Finally got around to looking at this PR.
This is something we should have done for a while, really appreciate the work!

File: index.md (hunk 1 above)
Comment (seanofw) [reply index.md:15]: If OpenTK 5 is released later this year, what else would you call OpenTK 3 then?  Do we support all of OpenTK 3, 4, and 5 at that point?

Comment (NogginBops) [reply index.md:15]: We would support OpenTK 3, and OpenTK 5. Basically, .net framework is going to live for a while so OpenTK 3 is also probably going to live for a while.

But sure, I think I would say "OpenTK 3 is for if you are stuck with .net framework" or something like that. But I guess using the term "legacy" is fine.

File: index.md (hunk 2 above)
Comment (seanofw) [reply index.md]: I've added updates to the tutorials links, so that both are included.

**Comment by seanofw** [comment]
All right, any other thoughts?  I'll keep updating this as long as there's anything else that's needed.

**Pull request merged**